app/controllers/staff/proposals_controller.rb

```diff
@@ -3,6 +3,8 @@ class Staff::ProposalsController < Staff::ApplicationController
 
   before_action :require_proposal, only: [:show, :update_state, :update_track, :update_session_format, :finalize]
   before_action :enable_staff_selection_subnav
+  skip_before_action :require_program_team, only: [:update_track, :update_session_format]
```

Comment:
I believe you can just change require_program_team to become require_reviewer instead.

Reply:
Oh, I guess nevermind on this as I didn't think about the fact you're undoing this from ProgramSupport.